app/test/specs/commentServiceSpec.js

```diff
@@ -0,0 +1,126 @@
+'use strict';
+
+describe('Test CommentService', function() {
+    var http, httpBackend, deferred, scope, commentService, comments, comment, answer, error;
+    var postCommentsUri = '/api/posts/post-key/comments';
+    comment = {text: 'text', post_key: 'post-key', id: 'comment-id'};
+
+    beforeEach(module('app'));
+
+    beforeEach(inject(function($http, $httpBackend, $q, $rootScope, CommentService) {
+        http = $http;
+        httpBackend = $httpBackend;
+        deferred = $q.defer();
+        scope = $rootScope.$new();
+        commentService = CommentService;
+        comments = [comment];
+        httpBackend.when('GET', 'main/main.html').respond(200);        
+        httpBackend.when('GET', 'home/home.html').respond(200);
+        httpBackend.when('GET', 'error/error.html').respond(200); 
+    }));
+
+    afterEach(function() {
+        answer = undefined;
+        error = undefined;
+        httpBackend.verifyNoOutstandingExpectation();
+        httpBackend.verifyNoOutstandingRequest();
+    });
+
+    describe('Test getComments', function() { 
+        beforeEach(function() {
+            spyOn(http, 'get').and.returnValue(deferred.promise);
+            commentService.getComments(postCommentsUri).then(
+                function success(response) {
+                    answer = response;
+                }, function err(response) {
+                    error = response;
+                }
+            );
+        });
+
+        it('Success case', function() {
```

Comment:
segue o padrão de nomes dos it's/describes:

https://github.com/eciis/web/blob/6811455da7a2e2e08fff333065fdc1df9509cf3f/app/test/specs/homeControllerSpec.js